src/app/Caches/Create/CreateCacheWizard.tsx

```diff
@@ -177,18 +275,80 @@ const CreateCacheWizard = (props) => {
         nextButtonText: 'Create Cache',
     };
 
-    const steps = [stepGettingStarted, stepEditCode];
+    const stepConfigure = {
+        name: 'Configuration',
+        steps: [
+            {
+                id: 3,
+                name: 'Basic',
+                component: (
+                    <ConfigurationBasic formConfigurationBasic={formConfigurationBasic} handleFormConfigurationBasic={handleFormConfigurationBasic} />
+                )
+            },
+            { id: 4, name: 'Cache Feature', component: <p>Cache Feature</p> },
+            { id: 5, name: 'Advanced options', component: <p>Advanced options</p> },
+        ]
+    };
+
+    const stepReview = {
+        id: 6,
+        name: 'Review',
+        component: (
+            <p>Review</p>
+        )
+    }
+
+    const steps = [stepGettingStarted,
+        ...(stateObj.showConfigurationStep ? [stepConfigure] : []),
+        ...(stateObj.showEditCodeStep ? [stepEditCode] : []),
+        ...(stateObj.showReviewStep ? [stepReview] : [])
+    ];
+
+    const CustomFooter = (
+        <WizardFooter>
+            <WizardContextConsumer>
+                {({ activeStep, goToStepByName, goToStepById, onNext, onBack, onClose }) => {
+                    return (
+                        <>
+                            <Button
+                                variant="primary"
+                                type="submit"
+                                onClick={(event) => getNextStep(event, activeStep, onNext)}
+                                isDisabled={!isFormValid}
+                            >
+                                {activeStep.name === t('caches.create.edit-config.nav-title') || activeStep.name === 'Review' ? "Create Cache" : "Next"}
+                            </Button>
+                            <Button
+                                variant="secondary"
+                                onClick={(event) => getPreviousStep(event, activeStep, onBack)}
+                                className={
+                                    activeStep.name === t('caches.create.getting-started.nav-title') ? "pf-u-hidden" : ""
```

Comment:
classe names are not used hardcoded this way. you need to use a patternfly token